### DISCUSSION TITLE
[Script request] ente io (alternative to Google Photos and Apple Photos)

### DISCUSSION THREAD

**Discussion by lappiesja** [community-scripts/ProxmoxVE#373] [Request script] [answered]
### Application Name

ente io

### Website

https://github.com/ente-io/ente

### Description

Ente is a service that provides a fully open source, end-to-end encrypted platform for you to store your data in the cloud without needing to trust the service provider. On top of this platform, we have built two apps so far: Ente Photos (an alternative to Apple and Google Photos) and Ente Auth (a 2FA alternative to the deprecated Authy).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
Not possible for now.

1st.) documentation is bad af. (Truly one of the worst docus I've ever seen in any script. Even though they look so "modern".)
2nd.) there are to many dependencies which are not defined
3th.) Ente doesn't really run under docker, or only damn poorly.

**Reply by lappiesja** [reply]
Thanks for your reply, really appreciate that. I hope that they will improve their documentation so that it will be possible to create a script in the future.

**Comment by iedame** [comment]
Hello,

Ente has revamped its documentation and now offers easy installation via Docker. Their documentation also provides details for non-Docker installations. Could you please reconsider this?

Thank you.

**Reply by MickLesk** [reply]
Link? I dont find something

**Reply by iedame** [reply]
I believe all the up to date documentation is available here: https://help.ente.io/self-hosting/installation/compose

**Reply by MickLesk** [reply]
Its Just only docker too?

**Reply by iedame** [reply]
There is also a manual installation page: https://help.ente.io/self-hosting/installation/manual looks like they expect those who will use the manual installation to use external S3, so also need MinIo installed if self hosting object storage: https://min.io/docs/minio/linux/operations/installation.html

**Reply by MickLesk** [reply]
Alright @tremor021 investigate?

**Reply by ghost** [reply]
Any update on this been wanting to try this

**Reply by MickLesk** [reply]
In Dev Repo

**Reply by anubis-genix** [reply]
Dev Repo means almost close to be released or are there still problems that need to be addressed?

**Reply by MickLesk** [reply]
Test it and Report in the issue (ProxmoxVED) ^^

**Comment by anubis-genix** [comment]
Thanks, wasn't aware of that VED repo
